packages/core/src/PlayerContextProvider.js

```diff
@@ -685,13 +703,10 @@ export class PlayerContextProvider extends Component {
     });
   }
 
-  handleMediaLoadedmetadata() {
+  handleMediaLoadedData() {
     if (this.media.currentTime !== this.state.currentTime) {
       this.media.currentTime = this.state.currentTime;
     }
-    this.setState(
-      state => (state.trackLoading === false ? null : { trackLoading: false })
-    );
```

Comment:
A few thoughts here:
1. Remember that `state.trackLoading` is being used to enforce currentTime while the track is loading in `handleMediaTimeupdate`
2. Ok but speaking of that... I am looking at [the diff where I introduced that](https://github.com/benwiley4000/cassette/commit/a1f99b0ee7acccec469d4eb23d675b5f74c88d8f#diff-85f3673c9affae062acd5232f1e0243bR567) and I honestly have no idea why I added this (I wish I had left a comment!). Is it even possible for the currentTime to change to something inconsistent with the current state before the metadata has loaded? Maybe we should totally remove that check and use the `trackLoading` state just for API reasons.
3. I think `canplaythrough` is too late for setting `trackingLoading: false`. If you want to change it to `canplay`, I could agree with that.

Reply:
1, 2 - I think `trackLoading` is really useful for users UI. If you want to have something like `trackLoadingState`, where 0 is not loading, 1 metadata, 2 canplay, it would be more flexible for users to decide when they want to stop displaying their loading UI, and you could check for loadingState 1 in timeupdate, so that logic stays the same. I can tell you however that I've tried these PR changes, and I didn't find any issue.
3 - Sure

Reply:
I hadn't considered exposing something like you're suggesting that essentially maps to the readyState. However that sounds a bit more advanced, so for now I'd say just go ahead with continuing to expose `trackLoading` and make it true when `canplay` fires. Thanks!!

Reply:
Done